packages/daemon/src/utils/aws.ts

```diff
@@ -0,0 +1,53 @@
+import { SendNotificationToDevice, Severity, WalletBalanceValue } from "../types";
+import { LambdaClient, InvokeCommand, InvokeCommandOutput } from '@aws-sdk/client-lambda';
+import { StringMap } from '../types';
+import getConfig from '../config';
+import logger from '../logger';
+import { addAlert } from "./alerting";
+
+export function buildFunctionName(functionName: string): string {
+  const { STAGE } = getConfig();
+  return `hathor-wallet-service-${STAGE}-${functionName}`;
+}
+
+/**
+ * Invokes this application's own intermediary lambda `OnTxPushNotificationRequestedLambda`.
+ * @param walletBalanceValueMap - a map of walletId linked to its wallet balance data.
+ */
+export const invokeOnTxPushNotificationRequestedLambda = async (walletBalanceValueMap: StringMap<WalletBalanceValue>): Promise<void> => {
+  const {
+    PUSH_NOTIFICATION_ENABLED,
+    WALLET_SERVICE_LAMBDA_ENDPOINT,
+    ON_TX_PUSH_NOTIFICATION_REQUESTED_FUNCTION_NAME,
+  } = getConfig();
+
+  if (PUSH_NOTIFICATION_ENABLED) {
+    logger.debug('Push notification is disabled. Skipping invocation of OnTxPushNotificationRequestedLambda lambda.');
+    return;
+  }
```

Comment:
```suggestion
  if (!PUSH_NOTIFICATION_ENABLED) {
    logger.debug('Push notification is disabled. Skipping invocation of OnTxPushNotificationRequestedLambda lambda.');
    return;
  }
```

Reply:
This is weird, but fixed in d8a39670dc0bf2fc0293eecc2bc3e7aa57721316